docs/reference/supported-environments.md

```diff
@@ -93,6 +93,7 @@ Requirements for the components can be seen below:
 | Optimize    | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.5.x                                                                                                                                                                   |
 | Connectors  | OpenJDK 21+  |                                                                                                                                                                                                                   |
 | Web Modeler | -            | PostgreSQL 13.x, 14.x, 15.x, 16.x or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x, 16.x                                                                                                                              |
+| Console     | -            | -                                                                                                                                                                                                                 |
```

Comment:
Was this added for completeness or are there additional items Console needs in this table?

Reply:
mainly for completeness might just update one part that the team suggested to add which was nodejs but I think for now at least we have it mentioned.

Reply:
Please remove this for now.